onnxruntime/core/session/inference_session.h

```diff
@@ -179,6 +179,14 @@ class InferenceSession {
     */
   common::Status Load(std::istream& model_istream);
 
+  /**
+    * Load an ONNX model.
+    * @param model_data Model data buffer
+    * @param model_data_len Model data buffer size
+    * @return OK if success.
+    */
+  common::Status Load(const void* model_data, size_t model_data_len);
```

Comment:
model_data_len should be in 'int' type ,not size_t. 
You're current implementation will silently discard data if it's larger than 2GB. It doesn't looks good.

Reply:
You're absolutely correct, that was stupid... It's fixed now.